src/components/credit.tsx

```diff
@@ -29,7 +29,7 @@ const Credit: FC<Props> = ({ format, credit }) =>
 				case Design.Media:
 					return <p css={styles}>{cred}</p>;
 				default:
-					return <> {cred}</>;
+					return <Fragment> {cred}</Fragment>;
```

Comment:
I don't think this is needed now that we have the Jest config updated.
```suggestion
					return <> {cred}</>;
```